variants/frontend-react/template.rb

```diff
@@ -8,6 +8,8 @@
 run "rails webpacker:install:react"
 run "rails generate react:install"
 
+run "yarn add --dev @testing-library/react @testing-library/jest-dom @testing-library/user-event"
```

Comment:
🐘 nity thing that could be worth doing in the future: having this in `test/template.rb`, to match how we've got `lint/template.rb`